contents/docs/features/events.md

```diff
@@ -40,26 +42,38 @@ You can have one or more filters.
 These refine the view to show just Events with a selected property:
 
 ![Event property filtering](../../images/02/Screenshot-2020-02-09-at-18.09.29.png)
+<br>
 
-## Autocapture event tracking
+## Autocapture Event Tracking
 
 PostHog has the capability to capture all front end events automatically from just a simple JS snippet.
 
 This means you do not need to add track(‘event’) to individual buttons, or parts of your product any more. The end result is zero maintenance of your event tracking, and less developer time spent on analytics. Focus on making product decisions and building cool features!
 
 The other advantage is that you won’t lose data. If you change your product a lot, you can always work backwards with your analytics.
 
-## Push-based event tracking
+#### Important Note on Autocapture
+
+While autocapture allows you to track the majority of general events on your website right out of the gate, it is important to note that, for security reasons, PostHog is very conservative regarding `input` tags. In order to prevent passwords or other sensitive data from being collected, very little data is collected from inputs with autocapture.
```

Comment:
Looks good. I'd mention we specifically capture `name`, `id` and `class` attributes

Reply:
Cool, will add that

Reply:
Should be fine now